detekt-rules/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/complexity/LongMethodSpec.kt

```diff
@@ -44,5 +45,41 @@ class LongMethodSpec : Spek({
 
             assertThat(subject.compileAndLint(code)).isEmpty()
         }
+
+        it("should not find too long method with params on newlines") {
+            val code = """
+                fun methodWithParams( // 4 lines
+                    param1: String,
+                    param2: String,
+                    @QueryValue startDate: String,
+                    @QueryValue endDate: String
```

Comment:
With this modification the code snippet actually compiles.
```suggestion
                    startDate: String,
                    endDate: String
```

Reply:
Can you check the tagged issue, please?

Reply:
I get it now. Thanks!